controllers/userController.js

```diff
@@ -175,17 +175,71 @@ const currentUser = async (req, res, next) => {
   ApiResponse.result(res, { user: data }, httpStatusCodes.OK);
 };
 
+const updateUser = async (req, res, next) => {
+  const { firstName, lastName, phone } = req.body;
+
+    const user = await Users.updateOne(
+      { _id: req.params.id },
+      { $set: {firstName, lastName, phone} }
```

Comment:
what if I only want to update firstName not lastName ? In this case If I send only firstName in req.body lastName and phone will be set as undefined. Please update.